routes/index.js

```diff
@@ -17,13 +17,15 @@ module.exports = (app) => {
       layout: 'main.handlebars',
     });
   });
-
-  app.get('/about', async (req, res) => {
-    res
-      .status(200)
-      .render('static/about', { title: 'About', layout: 'main.handlebars' });
```

Comment:
@neilvarunnaidu Please add layout here

Reply:
you also need to rebase your branch with master. Let me know in case you need help